src/sqlancer/GlobalState.java

```diff
@@ -89,8 +91,7 @@ public void setDatabaseName(String databaseName) {
         this.databaseName = databaseName;
     }
 
-    public boolean executeStatement(Query q) throws SQLException {
-        boolean logExecutionTime = getOptions().logExecutionTime();
+    public ExecutionTimer logExecution(Query q, boolean logExecutionTime) throws SQLException {
```

Comment:
Do we need to pass `logExecutionTime` here? I think the method can just use `getOptions()` instead, no?